### PR TITLE
Work in progress: anniversary blog post

### PR DESCRIPTION
**Pull request by bep**

### CODE REVIEW

**Comment by bep** [comment]
@digitalcraftsman I created a PR for this to get the preview. Please add to it (numbers? There are some other stats floating around that makes us look good). 

I created an outline, will add "my story" tomorrow.

@kaushalmodi @spf13 @regisphilibert and gang ... input/suggestions.

File: content/en/news/lets-celebrate-hugos-5th-birthday/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+date: 2018-07-04
+title: "Let’s celebrate Hugo’s 5th birthday"
```
Comment (kaushalmodi) [inline content/en/news/lets-celebrate-hugos-5th-birthday/index.md]: Do you want to use a straight-quote here: "Hugo’s" -> "Hugo's"?

File: content/en/news/lets-celebrate-hugos-5th-birthday/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+categories: [blog]
+author: bep
+---
+
+
+Gopher artwork by [Ashley McNamara](https://github.com/ashleymcnamara/gophers/) (licensed under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)). Inspired by [Renee French](https://reneefrench.blogspot.com/).
+
+Some notable sites.
+
+https://www.smashingmagazine.com/ https://vote.gov/ https://www.netlify.com https://gohugo.io/showcase/letsencrypt/ https://kubernetes.io/
```
Comment (kaushalmodi) [inline content/en/news/lets-celebrate-hugos-5th-birthday/index.md]: Should these sites be put as individual list items?

File: content/en/news/lets-celebrate-hugos-5th-birthday/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Some numbers:
+
+
+* 26800+ [stars](https://github.com/gohugoio/hugo/stargazers) on GitHub. This makes us currently the second most starred static site generator, according to [StaticGen.com](https://www.staticgen.com/)[^sgen]
+* 444+ [contributors](https://github.com/gohugoio/hugo/graphs/contributors)
+* 235+ [themes](http://themes.gohugo.io/)
```
Comment (kaushalmodi) [inline content/en/news/lets-celebrate-hugos-5th-birthday/index.md]: http -> https

File: content/en/news/lets-celebrate-hugos-5th-birthday/index.md (hunk 2 above)
Comment (bep) [reply content/en/news/lets-celebrate-hugos-5th-birthday/index.md]: I just dumped them in there... bokmarks for later. When I asked for input I meant on a more general level. This is a rough draft that I will finish tomorrow.

File: content/en/news/lets-celebrate-hugos-5th-birthday/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+date: 2018-07-04
+title: "Let’s celebrate Hugo’s 5th birthday"
+description: "How a side project became the fastest and one of the most popular framework for building websites."
```
Comment (moorereason) [inline content/en/news/lets-celebrate-hugos-5th-birthday/index.md]: Strike "the fastest and" and make "framework" plural.

```yaml
description: "How a side project became one of the most popular frameworks for building websites."
```

**Comment by digitalcraftsman** [comment]
I'll try to create a line chart that shows the rise of Hugo from release to release using the GitHub stars as metric.

Regarding content suggestions: static site generators became in vogue over last couple of years. Hugo was already there and took off. Just look at companies like Forestry and Netlify that emerged and build an infrastructure and tools around Hugo / Jekyll.

I'm not sure what @bep planned for tomorrow but it might be interesting to know why our lead developer jumped on this train. Why Go? Why an SSG like Hugo?

Furthermore, it would be interesting for a lot of people what @bep's vision is for the future of Hugo. What needs to be done to get v1.0 released?

Btw, I like the addition of the Hugo gopher to the artwork :smile:

**Comment by digitalcraftsman** [comment]
Here are two diagrams to illustrate Hugo's development in terms of GitHub stars and the number of themes:

themes and releases
github stars and releases

**Comment by bep** [comment]
@digitalcraftsman these are great, but I suspect that a time based graph would paint a more accurate picture. We at some point went from a 6 month to a 5 week release cycle, which obviously makes a different scale.

**Comment by digitalcraftsman** [comment]
I replaced the version numbers with the dates of the releases. Google spreadsheets should scale the release dates axis automatically.

number of themes and release dates
github stars and release dates

**Comment by regisphilibert** [comment]
This is a tight race but as of now Hugo is still number 3 “starry wise”. StaticGen is not dynamic so there is no way to really know at what their numbers will be today. Maybe Hugo will retake the lead right before they capture today’s stats but we can’t know...

Maybe rephrase to “this makes us one of the three most starred static site generator, according to...

**Comment by bep** [comment]
>Maybe rephrase to “this makes us one of the three most starred static site generator, according to...

I have removed that reference. And added more.

**Comment by regisphilibert** [comment]
That is one great post.

**Comment by digitalcraftsman** [comment]
Shall we spread the word?

**Comment by bep** [comment]
>Shall we spread the word?

I will ... thought I should wait until the americans in New york are up and running.

**Comment by digitalcraftsman** [comment]
The eastern part of the world is / was already awake. Hugo's popularity over there should compensate that :)

**Comment by bep** [comment]
I will Tweet now. It would be good if you could Reddit etc.?

**Comment by digitalcraftsman** [comment]
Reddit is already done. I could submit it to Hacker News as well.
